.github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml

```diff
@@ -0,0 +1,7 @@
+name: Feature Request
+description: Suggest a new idea for the project
+body:
+  - type: textarea
+    attributes:
+      label: "Description"
+      description: "Describe your feature"
```

Comment:
I think having a "why" section would be useful to understand the wider context. WDYT?

Reply:
Good idea, added